scripts/publish.mjs

```diff
@@ -164,6 +164,7 @@ async function previewChangelog() {
 
     await $`yarn install`;
     await $`yarn lerna publish --no-private --conventional-commits --create-release github`;
+    await $`yarn docs changelog`;
```

Comment:
If this is executed after the publish, when/where will this generated file (docs/src/data/log-md) be pushed to the repository, so that the docs are built with it?

Reply:
Ok so the release process will include an extra commit and push, made manually, right?

Reply:
Right, maybe later will add it also to a part of the script, as we are still gonna change it in Q2, the release process.